src/plugins/faultManagement/FaultManagementInspectorViewProvider.js

```diff
@@ -0,0 +1,72 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+import FaultManagementInspector from './FaultManagementInspector.vue';
+
+import Vue from 'vue';
+
+import { FAULT_MANAGEMENT_INSPECTOR, FAULT_MANAGEMENT_TYPE } from './constants';
+
+export default function FaultManagementInspectorViewProvider(openmct) {
+    return {
+        openmct: openmct,
+        key: FAULT_MANAGEMENT_INSPECTOR,
+        name: 'FAULT_MANAGEMENT_TYPE',
+        canView: (selection) => {
+            if (selection.length !== 1 || selection[0].length === 0) {
+                return false;
+            }
+
+            let object = selection[0][0].context.item;
+
+            return object && object.type === FAULT_MANAGEMENT_TYPE;
+        },
+        view: (selection) => {
+            let component;
+
+            return {
+                show: function (element) {
+                    component = new Vue({
+                        el: element,
+                        components: {
+                            FaultManagementInspector
+                        },
+                        provide: {
+                            openmct,
+                            selection
```

Comment:
Could you send in selection[0][0].context.item here instead of just the selection? That might simplify the Vue component

Reply:
hmm Actually not using it,  getting selection by `const selection = this.openmct.selection.get();`